pkgs/client/__tests__/fixtures.ts

```
@@ -0,0 +1,140 @@
+  status: FlowStepStatus.Failed,
+  error_message: 'Step failed',
+  failed_at: new Date().toISOString(),
+};
+
+// Helper function to emit events & advance timers
+export function advanceAndFlush(ms: number): void {
+  vi.advanceTimersByTime(ms);
+  // Flush any pending microtasks
+  return new Promise((resolve) => {
+    setImmediate(resolve);
+  });
```

Biome (1.9.4)

[error] The function should not return a value because its return type is void.

The function is here:

'void' signals the absence of value. The returned value is likely to be ignored by the caller.

(lint/correctness/noVoidTypeReturn)